runtime/firecrackeroci/vm.go

```diff
@@ -85,25 +85,26 @@ func WithVmLocalImageConfig(image containerd.Image) oci.SpecOpts {
 		}
 		spec.Process.Env = replaceOrAppendEnvValues(defaults, spec.Process.Env)
 		cmd := config.Cmd
-		spec.Process.Args = append(config.Entrypoint, cmd...)
+		config.Entrypoint = append(config.Entrypoint, cmd...)
+		copy(spec.Process.Args, config.Entrypoint)
```

Comment:
It doesn't look "better" than the original.

Reply:
Good catch. Not as concise, but probably more correct would be the following:
```go
spec.Process.Args = append([]string{}, config.Entrypoint...)
spec.Process.Args = append(spec.Process.Args, cmd...)
```
@kzys, thoughts?

Reply:
That looks better. We shouldn't modify `config.Entrypoint` to just make linters happy. 👍

Reply:
Agreed.